cloudformationTemplates/startChatContactAPI/README.md

```diff
@@ -1,83 +1,137 @@
+    | eu-central-1 (Frankfurt) | [![Launch Stack](https://cdn.rawgit.com/buildkite/cloudformation-launch-stack-button-svg/master/launch-stack.svg)](https://eu-central-1.console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=startChatContactAPI&templateURL=https://s3-eu-central-1.amazonaws.com/eu-central-1.start-chat-contact-api-cfn/cloudformation.yaml) |
+    | eu-west-2 (London) | [![Launch Stack](https://cdn.rawgit.com/buildkite/cloudformation-launch-stack-button-svg/master/launch-stack.svg)](https://eu-west-2.console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=startChatContactAPI&templateURL=https://s3-eu-west-2.amazonaws.com/eu-west-2.start-chat-contact-api-cfn/cloudformation.yaml) |
+    | ap-southeast-1 (Singapore) | [![Launch Stack](https://cdn.rawgit.com/buildkite/cloudformation-launch-stack-button-svg/master/launch-stack.svg)](https://ap-southeast-1.console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=startChatContactAPI&templateURL=https://s3-ap-southeast-1.amazonaws.com/ap-southeast-1.start-chat-contact-api-cfn/cloudformation.yaml) |
+
+
+4. Update the CloudFormation template with details from your connect instance.
+    - Fetch the Contact ID and Instance ID from your Connect instance by going to your contact flows and selecting **Sample queue customer** and clicking on **Show additional flow information** on top left. 
+    - `contactFlowId` \ `instanceId`: You can find the contact flow id and instance id when viewing a contact flow. For example, if the arn for your flow is 'arn:aws:connect:us-west-2:123456789012:instance/11111111-1111-1111-1111-111111111111/contact-flow/22222222-2222-2222-2222-222222222222', the contact flow id is '22222222-2222-2222-2222-222222222222' and instance id is 11111111-1111-1111-1111-111111111111
+
+        ![Contact Flow Id's](images/updateContactFlowIds.png)
+
+    - Go to your [CloudFormation stack](https://us-west-2.console.aws.amazon.com/cloudformation/home) by clicking on Stacks and then startChatContactAPI on the left hand side. Update the Contact Flow ID and the Instance ID in your cloud formation stack
+    ![Update Cloud Formation Stack](images/updateCloudFormationStack.png)
+    -   These are added as Lambda environment variables.
 
-Note: you will want to show the widget only when there is a conversation in progress. If the widget is visible when there is no conversation, you will just see a loading spinner.
+## Getting started with Pre Built Widget
 
-1. In your website's html code, import the 'amazon-connect-chat-interface.js' file from this repo.
+You can refer to the `widgetIndex.html` file in this repo to see an example of how to use the widget.
 
+1. In your website's html code, import the 'amazon-connect-chat-interface.js' file from this repo. Also import jQuery
     ```html
```

Comment:
this code block is double indented.

Reply:
It's rendering fine on my preview :). What do you see? 
![Screen Shot 2020-04-15 at 7 09 59 PM](https://user-images.githubusercontent.com/9209695/79406847-f0a7eb00-7f4c-11ea-8939-19975e3010f0.png)

Reply:
even in the picture you posted the two script tags are indented once already inside of the code block. you have extra whitespace.